po/pl.po

```diff
@@ -0,0 +1,441 @@
+msgstr "Wybiera kolor diody LED"
+
+#: data/ui/LedDialog.ui:90 data/ui/LedDialog.ui:256
+msgid "Choose a brightness for the LED"
+msgstr "Wybiera jasność diody LED"
+
+#: data/ui/LedDialog.ui:104 data/ui/LedDialog.ui:269
+msgid "Brightness"
+msgstr "Jasność"
+
+#: data/ui/LedDialog.ui:135 data/ui/LedDialog.ui:300
+msgid "Choose an effect rate for the LED"
+msgstr "Wybiera tempo efektu diody LED"
+
+#: data/ui/LedDialog.ui:152 data/ui/LedDialog.ui:317
+#: data/ui/ResolutionsPage.ui:105
+msgid "Hz"
+msgstr "Hz"
+
+#: data/ui/LedDialog.ui:174 data/ui/LedDialog.ui:339
+msgid "Effect rate"
+msgstr "Tempo efektu"
+
+#: data/ui/LedDialog.ui:211
+msgid "Pick a Color for the LED"
+msgstr "Kolor diody LED"
+
+#: data/ui/LedDialog.ui:225
+msgid "Color"
+msgstr "Kolor"
+
+#: data/ui/LedDialog.ui:378
+msgid "This LED is off"
+msgstr "Ta dioda LED jest wyłączona"
+
+#: data/ui/Menus.ui:7
+msgid "_About"
+msgstr "_O programie"
+
+#: data/ui/Menus.ui:11
+msgid "_Quit"
+msgstr "Za_kończ"
+
+#: data/ui/MousePerspective.ui:44
+msgid "Failed to commit changes to the device"
+msgstr "Wysłanie zmian do urządzenia się nie powiodło"
+
+#: data/ui/MousePerspective.ui:102
+msgid "Select another profile"
+msgstr "Wybiera inny profil"
+
+#: data/ui/MousePerspective.ui:127
+msgid "Commit the changes to the device"
+msgstr "Wysyła zmiany do urządzenia"
+
+#: data/ui/MousePerspective.ui:171
+msgid "Click to switch to another profile"
+msgstr "Przełącza na inny profil"
+
+#: data/ui/MousePerspective.ui:186
+msgid "Add profile"
+msgstr "Dodaj profil"
+
+#: data/ui/MousePerspective.ui:190
+msgid "Add a new profile to the device"
+msgstr "Dodaje nowy profil do urządzenia"
+
+#: data/ui/OptionButton.ui:9
+msgid "Open the configuration dialog"
+msgstr "Otwiera okno konfiguracji"
+
+#: data/ui/ProfileRow.ui:16
+msgid "Remove this profile from the device"
+msgstr "Usuwa ten profil z urządzenia"
+
+#: data/ui/ResolutionRow.ui:23
+msgid "Remove this resolution from the profile"
+msgstr "Usuwa tę rozdzielczość z profilu"
+
+#. Translators: this is used to indicate the active resolution.
+#: data/ui/ResolutionRow.ui:65
+msgid "Active"
+msgstr "Aktywna"
+
+#: data/ui/ResolutionRow.ui:105
+msgid "Set this resolution's DPI"
+msgstr "Ustawia DPI tej rozdzielczości"
+
+#. Translators: the rate at which the device reports movement.
+#: data/ui/ResolutionsPage.ui:43
+msgid "Report rate"
+msgstr "Tempo zgłaszania"
+
+#: data/ui/ResolutionsPage.ui:60
+msgid "Change the profile's report rate"
+msgstr "Zmienia tempo zgłaszania profilu"
+
+#. Translators: the sensitivity of the device's sensor.
+#: data/ui/ResolutionsPage.ui:125
+msgid "Sensitivity"
+msgstr "Czułość"
+
+#: data/ui/ResolutionsPage.ui:174
+msgid "Add a new resolution to the profile"
+msgstr "Dodaje nową rozdzielczość do profilu"
+
+#: data/ui/WelcomePerspective.ui:20
+msgid "Welcome"
+msgstr "Witamy"
+
+#: data/ui/WelcomePerspective.ui:38 data/ui/WelcomePerspective.ui:89
+msgid "Select a device to configure it"
```

Comment:
I would like to translate the label and the tooltip differently. Please add context to one of them:

https://wiki.gnome.org/TranslationProject/DevGuidelines/Translation%20contexts